Adyen/Analytics/AnalyticsProvider/AnalyticsProvider.swift

```diff
@@ -126,7 +126,11 @@ internal final class AnalyticsProvider: AnalyticsProviderProtocol {
         
         // as per this call's limitation, we only send up to the
         // limit of each event and discard the older ones
-        var request = AnalyticsRequest(checkoutAttemptId: checkoutAttemptId)
+        let platform = configuration.context.platform.rawValue
+        var request = AnalyticsRequest(
+            checkoutAttemptId: checkoutAttemptId,
+            platform: platform
```

Comment:
what about making the platform parameter type an enum instead of String? So you don't have to get the `rawValue` and it also reduces potential mistakes

Reply:
Could be good but would either need to expose the existing enum which this struct shouldn't know about or create a new one which feels redundant duplication :D
We are setting directly from the enum so it should be fine